### DISCUSSION TITLE
Auto refresh not working

### DISCUSSION THREAD

**Discussion by AaronPenton** [parcel-bundler/parcel#7019] [Q&A]
Hi there. I'm brand new to Parcel (or any bundler for that matter). I'm having several issues. The main being auto refresh (hot module replacement?) works intermittently and sometimes not at all. I was using version 1, and had this in my package.json:

```
"scripts": {
    "dev": "parcel src/*.index.html",
    "build": "parcel build src/*.index.html"
  },
```
I have 2 html files - index.html and snake.html. When changing the html files - something very simple, like changing `<p>test</p>` to `<p>test2</p>` , sometimes one would refresh and the other wouldn't, then vice versa. Same with when I changed a SASS file - one time index would update and snake wouldn't, and vice versa. No rhyme or reason as to why.

Then I realized I was using Parcel 1, so I updated to v2, and now nothing auto refreshes. The fact that it would work and then not work when all I was doing was updating a character in the HTML is just baffling to me. I'm ready to give up on this but I really don't want to.

I followed Kevin Powell's YT tutorial (he was using V1). The first sign of trouble was when I got to this in the command line within VSCode:

`sass --watch src/scss:dist/css`

I get "no such file or directory.". My command line is pointing to the correct folder. However I'm not sure this is an issue, as the SCSS is definitely compiling on save and the dist files are correctly being created. I know this is a manual refresh shows my changes correctly.

Can anyone help? Apologies for my newbness. I so badly want this to work!

**Comment by BryanCanCode313** [comment]
I really hope someone assist with this thread question. This very close to what my problem is
